### DISCUSSION TITLE
multiple dose protocol

### DISCUSSION THREAD

**Discussion by AParisotto** [Open-Systems-Pharmacology/Forum#867] [Questions & Problems] [answered]
Hello everyone
what pharmacokinetic parameters should i evaluate using an oral multiple administration protocol?
and what does each mean?

**Comment by StephanSchaller** [comment]
Dear @AParisotto ,

what do you mean, exactly?

Generally, for PO, AUC (Area under Curve), Cmax (maximum concentration) and Tmax (Time of maximum concentration) are useful metrics.

Best,
Stephan

**Comment by AParisotto** [comment]
the pharmacokinetic parameters after several administrations are those below.
which ones should be evaluated?

![Capturar](https://user-images.githubusercontent.com/53835113/107778421-d41e4000-6d22-11eb-89c2-e3702bed2a55.PNG)

AUC_inf_tD1
Area under the concentration vs. time curve from the first data point extrapolated to infinity (further administrations are not
considered!)
AUC_inf_tD1_n
oArmrea under the concentration vs. time curve from the first data point extrapolated to infinity (further administrations are not
considered!) normalized to dose, with dose in drug mass per body weight
....tDi-tDj 
Respective PK parameter from the administration time of the first dose until the administration time of the second dose
tDlast-tDEnd 
Respective PK parameter following the last application
....tDlast-1-
tDlast
Respective PK parameter in the intervall between the (last -1) application and the last application
C_trough_dDi 
Trough concentration just before the i-th dose is administered
C_trough_dlast 
Trough concentration just before the last dose is administered

**Comment by StephanSchaller** [comment, marked as answer]
It depends on what you need, but per default, the choice would be the PK Properties in steady-state, so this one
![image](https://user-images.githubusercontent.com/25477643/107782180-0d69a680-6d49-11eb-8340-c7c895d5f321.png)

(and accordingly the same for AUC and Cmax)
